### ISSUE TITLE
Rain goes through transparent blocks

### ISSUE THREAD

**Issue by Barteks2x** [Barteks2x/CubicChunks#45]

**Comment by Arzte** [comment]
Transparent blocks like glass?

**Comment by Niko-sk2x** [comment]
Yes. In the current state using light heightmap for precipitation heightmap is the simplest way to get some kind of normal rain.

**Comment by Arzte** [comment]
Would it be better to check for transparent blocks then not have rain below that, or to make a new hightmap for rain?

**Comment by Niko-sk2x** [comment]
How would you check if you are under a transparent block *if that block may not be loaded*?

It will need a separate heightmap, so that's why I'm leaving it for later when I have memory efficient fully working heightmap for sunlight.

**Comment by Arzte** [comment]
Ohh, ok. Thanks for explaining it to me.
